onnxruntime/core/optimizer/qdq_transformer/selectors_actions/shared/utils.cc

```diff
@@ -35,7 +35,6 @@ static const OpVersionsAndSelector::OpVersionsMap GetMiscOpVersionsMap() {
           {"Transpose", {}},
           {"MaxPool", {12}},
           {"Resize", {}},
-          {"Split", {}},
```

Comment:
Note: This was previously using the DropQDQNodeGroupSelector which only checks that input[0] and output[0] have matching quantization params. We need all outputs to match the input.